src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -364,6 +783,10 @@ public void binaryContentGuaranteeHeadersContentType(final String uri) {
                         "the request entity must reject request if it cannot guarantee all of the response headers " +
                         "required by the LDP-NR interaction model in this specification.",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final String location = createExternalBinary(uri, info.getId(), "proxy", "text/plain");
+
+        doGet(location).then().header("Content-Type", "text/plain");
```

Comment:
Can probably merge this test with the previous.

Reply:
Sure, that seems reasonable, there is a lot of redundancy at the moment